### DISCUSSION TITLE
Feature Request - Ticket closed email template

### DISCUSSION THREAD

**Discussion by empty88** [osTicket/osTicket#6250] [Ideas]
How about a separate email template for changing the ticket status from open to solved/closed.

I would like to inform the user that the ticket is closed.

**Comment by ntozier** [comment]
You can use a canned response to do something like this now.
Select the canned response and select closed.  post the reply.

**Reply by MarcBanyard** [reply]
The following pull request does this and will automatically close answered tickets after X days.

https://github.com/osTicket/osTicket/pull/6445
